apps/ledger-live-mobile/src/screens/SyncOnboarding/LanguageSelect.tsx

```diff
@@ -120,16 +118,21 @@ const LanguageSelect = ({ device, productName }: Props) => {
 
   return (
     <Flex>
-      <Button
-        type="main"
-        outline
-        size="small"
-        Icon={ChevronBottomMedium}
-        iconPosition="right"
-        onPress={handleLanguageSelectOnPress}
-      >
-        {currentLocale.toLocaleUpperCase()}
-      </Button>
+      <TouchableOpacity onPress={handleLanguageSelectOnPress}>
+        <Flex
+          flexDirection="row"
+          alignItems="center"
+          height={"32px"}
+          pl="12px"
+          pr={3}
+          bg="opacityDefault.c10"
+          borderRadius="50"
+        >
+          <Text mr="2px">{currentLocale.toLocaleUpperCase()}</Text>
```

Comment:
use the spacing values
```suggestion
          <Text mr={1}>{currentLocale.toLocaleUpperCase()}</Text>
```